docs/source/example_notebooks/dowhy_causal_discovery_example.ipynb

```diff
@@ -595,7 +651,7 @@
       "Estimand name: frontdoor\n",
       "No such variable found!\n",
       "\n",
-      "Causal Estimate is -0.013018300750143652\n"
+      "Causal Estimate is -0.0130183007501401\n"
```

Comment:
what is the conclusion of the analysis? do the estimates differ substantially or not?